Chapter index (#11)
UtrechtUniversity/dataprivacyhandbook: CONTRIBUTING.md

## Before you contribute
Before you contribute, the following points are important to take into account:
1. Please read through our [Code of Conduct](https://github.com/UtrechtUniversity/dataprivacyhandbook/blob/main/CODE_OF_CONDUCT.md) and make sure you familiarize yourself with its contents. We expect all of our contributors to comply with the terms specified there.

> 2. At the moment, all contributions go through our GitHub repository, https://github.com/UtrechtUniversity/dataprivacyhandbook/. So, in order for you to be able to contribute, you first have to [create a GitHub account](https://github.com/join). **In order to be a collaborator in this repository, the UtrechtUniversity organization requires to use [2-factor authentication for your GitHub account](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)**. <br>

Reviewer: ```suggestion
2. At the moment, all contributions go through our GitHub repository, https://github.com/UtrechtUniversity/dataprivacyhandbook/. In order for you to be able to contribute, you first have to [create a GitHub account](https://github.com/join). **Note, the UtrechtUniversity organization on GitHub requires to use [2-factor authentication for your GitHub account](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)**. <br>
```
